src/argilla/server/contexts/accounts.py

```diff
@@ -57,7 +50,7 @@ def delete_user_workspace(db: Session, user_workspace: UserWorkspace):
 
 
 def get_workspace_by_id(db: Session, workspace_id: UUID):
-    return db.query(Workspace).get(workspace_id)
+    return db.get(Workspace, workspace_id)
```

Comment:
These `db.query(...).get()` calls was causing a warning and asking to replace them with new `session.get`.

Reply:
Also, it seems to be more efficient

Reply:
More efficient like on number of characters in the code or a more efficient db query?

Reply:
I can imagine some cache improvement mechanisms when using a get-by id instead of a more generic query. But not sure if really are done this kind of optimization.